protobuf/v1/nexus.proto

```diff
@@ -327,6 +329,10 @@ message RebuildHistoryRequest {
   string uuid = 1;  // uuid of the nexus
 }
 
+message RebuildHistoryListResponse {
+  repeated RebuildHistoryResponse histories =1; // List of rebuild histories from all nexus.
```

Comment:
RebuildHistoryResponse will have nexus and record vec

Reply:
I know, but then we still need to pick the each and put it in the right nexus rebuild. If we also have a map with nexus as key it will be much simpler.